UI/Steps/LoginPagesSteps.cs

```diff
@@ -11,11 +11,11 @@ public class LoginPagesSteps
     {
         private static LoginPage loginPage = new LoginPage();
 
-        public static MainPage Login(string username, string password, int timeSeconds)
+        public static MainPage Login(string username, string password)
         {
-            loginPage.EnterUsername(username, timeSeconds);
-            loginPage.EnterPassword(password, timeSeconds);
-            loginPage.ClickToLoginButton(timeSeconds);
+            loginPage.EnterUsername(username);
```

Comment:
As this page functions return Page then you can made it via
loginPage.EnterUsername(username).EnterPassword(password).ClickToLoginButton()
Same in functions below